ios/FluentUI/Navigation/SearchBar.swift

```diff
@@ -454,7 +455,9 @@ open class SearchBar: UIView {
         }
 
         searchTextFieldBackgroundView.addSubview(leadingView)
-        let leadingViewSize = leadingView.sizeThatFits(searchTextFieldBackgroundView.frame.size)
+        let leadingViewRenderSize = CGSize(width: searchTextFieldBackgroundView.frame.size.width - (Constants.searchIconImageViewDimension + Constants.searchIconInset + Constants.searchTextFieldLeadingInset + Constants.searchTextFieldInteractionMinWidth + Constants.clearButtonLeadingInset + Constants.clearButtonWidth + Constants.clearButtonTrailingInset),
```

Comment:
Would it make sense to compute all the constant values in a variable for readability?